components/namespace-lister/base/network_policy_allow_to_apiserver.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-to-apiserver
+  namespace: namespace-lister
+spec:
+  podSelector:
+    matchLabels:
+      apps: namespace-lister
+  policyTypes:
+  - Egress
+  egress:
+  - to:
+    - podSelector:
+        matchLabels:
+          app: openshift-kube-apiserver
+      namespaceSelector:
+        matchLabels:
+          kubernetes.io/metadata.name: openshift-kube-system
```

Comment:
On `kflux-prd-rh02` I see the pods with the `app: openshift-kube-apiserver` label in the `openshift-kube-apiserver` namespace, not `openshift-kube-system`.

Reply:
great catch, that was a typo. fixed in 11777aca2bdb17c34904c0a45dd831d2b81ad62d